### PR TITLE
Start on scene parsing

### PR DESCRIPTION
**Pull request by ravertass**

### CODE REVIEW

File: PixelHunter1995/SceneParser.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+using System.Xml;
+
+namespace PixelHunter1995
+{
+    class SceneParser
+    {
+        public static void ParseSceneXml(String sceneXmlPath)
```
Comment (Hattes) [inline PixelHunter1995/SceneParser.cs:8]: Perhaps I'm thinking ahead. But I imagine this function should just return an object "Scene" containing all the data. 

And kinda related, should we ever write the xml when the scene gets deloaded, and it's been modified? Or should we just keep these objects in memory? My hunch is we shouldn't write it, but I guess it might help debugging.

File: PixelHunter1995/SceneParser.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (Hattes) [inline PixelHunter1995/SceneParser.cs]: Why do we need all this?

Comment (ravertass) [reply PixelHunter1995/SceneParser.cs]: Visual Studio just magically added this to my file when I create a new class... I can try removing it and see what happens.

Comment (Hattes) [reply PixelHunter1995/SceneParser.cs]: Det ante mig!

File: PixelHunter1995/SceneParser.cs (hunk 1 above)
Comment (ravertass) [reply PixelHunter1995/SceneParser.cs:8]: Yes, this function returning a scene object seems right.

I don't think we should ever modify the scene XML. Keeping objects in memory seems more correct. I'm not even sure if we should modify the scene objects. In Hundens Vagga I keep a "world state" object, containing info about what has happened in the world (e.g. if a door is unlocked). Since some state is not really related to a certain scene (e.g. "the electricity is on"), I think that will be easier. Then, scenes can contain data about how it will look when the world state has been altered.

But I haven't really thought this through, more than when writing Hundens Vagga. Perhaps we can come up with a better way to do it for this game, which after all will be more complex.

Saving the game in Hundens Vagga simply means writing the inventory items, the world state and the current room to a file. An example save file looks like this:
```
letter,mainkey,flashlight
SUBTITLES_ON:False,st_frozen_melted:False,st_shovel_taken:False,st_crawlspace_open:False,st_crawlspace_illuminated:False,st_water_running:False,st_window_open:False,st_handle_taken:False,st_maindoor_open:False,st_doormat_moved:True,st_mainkey_taken:True,st_flashlight_taken:True,st_typewriter_room_open:False,st_maps_open:False,st_notebook_taken:False,st_toykey_taken:False,st_wrench_taken:False,st_electricity_on:False,st_saucepan_taken:False,st_drawers_open:False,st_fuse_taken:False,st_microwave_open:False,st_typewriterkey_taken:False,st_toys_open:False,st_treasure_dug:False,st_musicbox_taken:False
car
```

File: PixelHunter1995/SceneParser.cs (hunk 2 above)
Comment (ravertass) [reply PixelHunter1995/SceneParser.cs]: Seems most of that was completely unnecessary, so I removed it.

File: PixelHunter1995/PixelHunter1995.csproj [hunk 3]
```diff
@@ -53,11 +54,17 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Content Include="Content\Images\Background1.jpg" />
+    <Content Include="Content\Sounds\495674__jack-urbanski__vibraphone-chord.wav" />
+    <Content Include="Content\Sounds\92734__tj-mothy__slow-sad-tones.wav" />
```
Comment (Hattes) [inline PixelHunter1995/PixelHunter1995.csproj:59]: Why are these here now when they weren't before?

**Review by Hattes — APPROVED** [review]

File: PixelHunter1995/PixelHunter1995.csproj (hunk 3 above)
Comment (ravertass) [reply PixelHunter1995/PixelHunter1995.csproj:59]: I added some stuff to the solution explorer in Visual Studio. Stuff that exist in the already present `Content` folder, and which I thought was nice to be able to see there.

This doesn't change anything else but the view in Visual Studio. However, further down in this file, you can see that I made the scene XML file be copied to the build directory when building, so it can be read during run-time.
